worksheet_B/Mandelbrot.cpp

```diff
@@ -12,45 +14,116 @@ int main()
 	CImgDisplay display(image, "Mandelbrot Set");
 
 	// Minimum and maximum coordinates for the fractal
-	const double minX = -2, maxX = 1, minY = -1.5, maxY = 1.5;
+	double minX = -2, maxX = 1, minY = -1.5, maxY = 1.5;
+	bool Render = true;
 
-	// Generate the image
-	for (int pixelY = 0; pixelY < image.height(); pixelY++)
+	// Main loop
+	while (true)
 	{
-		// TODO: Map the y coordinate into the range minY to maxY
-		//double y0 =
-
-		for (int pixelX = 0; pixelX < image.width(); pixelX++)
+		while (Render)
```

Comment:
You set `Render = false` at the end of the loop, so unless I'm missing something, it only ever executes once. Why not make it an `if` statement?